bonita/license.md

```diff
@@ -1,5 +1,6 @@
-Bonita BPM image includes two parts :
+Bonita image includes two parts :
 
--	Bonita BPM Engine under [LGPL v2.1](https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html)
+-	Bonita Engine under [LGPL v2.1](https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html)
+
+-	Bonita Portal under [GPL v2.0](http://www.gnu.org/licenses/old-licenses/gpl-2.0.html)
 
```

Comment:
Now `markdownfmt` is only complaining about this single extra blank line. :+1:

Reply:
Ok thanks, I've reverted this commit.